lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -273,7 +273,7 @@ class TestFieldOverrideSplitPerformance(FieldOverridePerformanceTestCase):
     __test__ = True
 
     # TODO: decrease query count as part of REVO-28
-    QUERY_COUNT = 33
+    QUERY_COUNT = 30
```

Comment:
This and other query count changes are because I moved the enrollment lookup below the "does user have staff access" check.